tests/testthat/test-get_code.R

```diff
@@ -653,3 +650,19 @@ testthat::test_that("get_call data call is returned when data name is provided a
     )
   )
 })
+
+testthat::test_that(
+  "get_code with datanames can extract the code when function creates an object which is used only on rhs",
```

Comment:
Please refine this description, it does not convey what case is tested.

Reply:
09b02dda70eb153ffa97f917956001d381451f17